scenarios/sanity_check.yml

```diff
@@ -0,0 +1,23 @@
+# This scenario similuates a minimal network comprising only a single validator
+# running one application generating a small number of transactions.
+
+# The name of the scenario
+name: Sanity Check for Main
```

Comment:
It seems to me that this scenario is almost identical to `small.yaml `we already have

Reply:
It is `single.yml` without any node.
The purpose of this is to check, if the network would function as expected after addition of static validators.
This would be the minimum required test as agreed in our call, to be expanded later to include other tests.